### PR TITLE
[7.4-only] LPS-121957 Migrate v2 usages of clay:vertical-card in asset-browser-web

### PR DESCRIPTION
**Pull request by julien**
In this change, we replace <clay:vertical-card-v2 />(clay 2.x: Metal+soy) with
<clay:vertical-card /> (clay 3.x: React)

**Test plan**

- Fetch this branch
- Deploy the asset-browser-web and frontend-taglib-clay modules
- Navigate to  **Site Builder** > **Collections**
- Create a new collection
- When selecting an item, in the modal window, change the display style
to "Cards"

**Additional notes**

Please take into account that currently there is [an open issue](https://issues.liferay.com/browse/LPS-123518)
with cards in this module (discussed with @liferay-echo)

For the moment, this submitting this PR as a draft for internal review with @carloslancha and @markocikos before sending it to @liferay-echo

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by julien** [comment]
Rebased with master

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 import com.liferay.frontend.taglib.clay.servlet.taglib.soy.VerticalCard;
 import com.liferay.frontend.taglib.clay.servlet.taglib.util.LabelItem;
+import com.liferay.petra.string.StringPool;
```
Comment (markocikos) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java:19]: A bit of leftover code?

**Review by markocikos — COMMENTED** [review]
LGTM, apart from the trailing line.

In the same way as https://github.com/liferay-frontend/liferay-portal/pull/502, we should wait until https://github.com/liferay-frontend/liferay-portal/pull/503 is merged so we can recommend using `<clay:navigation-tag>` instead of vertical tag with `interactive` class.

**Comment by carloslancha** [comment]
Yep, same case than https://github.com/liferay-frontend/liferay-portal/pull/502

I'd not wait for `clay:navigation` to be merged, this can be forwarded and just let the teams know that when we merge the other tag they can change the use if they want.

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java (hunk 1 above)
Comment (julien) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java:19]: Thanks let me check that, I think it's a left over from the rebase.

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by julien** [comment]
Hi @john-co and @PablitoBonito,
 
I this PR we're replacing `<clay:vertical-card-v2 />` with `<clay:vertical-card />` and looking at the poshi summary and reports, it doesn't seem that these errors are related. But could you confirm when you have a change?

Thanks

**Comment by john-co** [comment]
@PablitoBonito , @zhangyan0701, can one of you please take a look?

**Comment by john-co** [comment]
ci:test:bundle

**Comment by zhangyan0701** [comment]
**The failures are unrelated:**

- LocalFile.ContentPagesWithMapping#ViewMappableFieldsAreGroupedByFieldset is a known Test Fix: [LRQA-63190](https://issues.liferay.com/browse/LRQA-63190)
- LocalFile.AssetPublisherShowFullContent#AddAssetTypesViaAPAndAddRespectivePortletsToSamePage is a regression that also occurs on master upstream: [LPS-123708](https://issues.liferay.com/browse/LPS-123708)
- modules-integration-mysql57-jdk8/0 should be fluke that related to ES7: [LRQA-62694](https://issues.liferay.com/browse/LRQA-62694)

/cc @john-co @PablitoBonito

**Comment by julien** [comment]
thanks for the help @zhangyan0701 

(/cc @carloslancha @markocikos)

**Comment by carloslancha** [comment]
@julien do we need to forward this to Brian or to the team that owns this app?

**Comment by julien** [comment]
@carloslancha this is also for @liferay-lima :)

**Comment by julien** [comment]
Forwarded to https://github.com/liferay-lima/liferay-portal/pull/1082

**Comment by julien** [comment]
Sorry that was for @liferay-echo, forwarded here https://github.com/liferay-echo/liferay-portal/pull/2958

**Pull request closed without merging**